doc/language_reference/language_reference.md

```diff
@@ -426,6 +432,9 @@ vardecl_term ::= "var" var_name
 
 match_term   ::= "match" "(" expr ")" "{" match_clause (,match_clause)*"}"
 match_clause ::= pattern "->" expr
+
+lambda_term  ::= "function" "(" [expr ("," expr)*] ")" [":" simple_type_spec] expr
```

Comment:
no braces required here?

Reply:
Good point. I don't think they are strictly necessary, but we may want to requite them for consistency.

Reply:
fixed this